### PR TITLE
chore: Added unit tests for FileSignatureWaiversImpl

### PR DESCRIPTION
**Pull request by derektriley**
**Description**:
Adds unit tests for FileSignatureWaiversImpl. Brings coverage to 100%.

**Related issue(s)**:
#13191 

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileSignatureWaiversImplTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.file.impl.test.handlers;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
```
[the remark this answers is not in the document]
Comment (povolev15) [reply hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileSignatureWaiversImplTest.java:19]: +1

[the remark this answers is not in the document]
Comment (derektriley) [reply hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileSignatureWaiversImplTest.java]: Updated with latest commit.

**Review by thomas-swirlds-labs — APPROVED** [review]
LGTM. Thanks @derektriley

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM ! Thanks @derektriley

**Pull request merged**